wallet/src/Pos/Wallet/Web/Methods/Payment.hs

```diff
@@ -166,7 +167,7 @@ sendMoney SendActions{..} passphrase moneySource dstDistr = do
           let addrMeta =
                   fromMaybe (error "Corresponding adress meta not found")
                             (fst <$> find ((== addr) . snd) metasAndAdrresses)
-          sk <- getSKByAddress passphrase addrMeta
+          sk <- getSKByAddress (ShouldCheckPassphrase False) passphrase addrMeta
```

Comment:
And again, this is safe because we already did the `checkPassMatches passphrase rootSk`